### PR TITLE
[Meetup Project] Order Details: Support contacting customers through WhatsApp or Telegram

### PR DESCRIPTION
**Pull request by hichamboushaba**
### Description
This PR adds support for contacting customers using WhatsApp or Telegram when they are installed from the order details.

### Testing instructions
**Please repeat for WhatsApp and Telegram**
##### TC1: app installed
1. Ensure WhatsApp is installed
2. Create an order (or edit one) to ensure it has a billing phone number.
3. Open the order details in the app.
4. Show the billing information.
5. Click on the menu button next to the phone number.
6. Ensure an entry for the app is shown (Contact using {app}).
7. Click the button, and confirm it works as expected.

##### TC2: app not installed
1. Repeat 1 to 5 from TC1
2. Confirm the entry for the app is not shown.

### Images/gif

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-android/milestone/196">17.6</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false"><p>This PR contains changes to Tracks-related logic. Please ensure (<strong>author and reviewer</strong>) the following are completed:</p>

<ul>
  <li>The tracks events must be validated in the Tracks system.</li>
  <li>Verify the internal Tracks spreadsheet has also been updated.</li>
  <li>Please consider registering any new events.</li>
  <li>The PR must be assigned the <strong>category: tracks</strong> label.</li>
</ul></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/details/views/OrderDetailCustomerInfoView.kt [hunk 1]
```diff
@@ -311,6 +315,34 @@ class OrderDetailCustomerInfoView @JvmOverloads constructor(
             true
         }
 
+        if (ActivityUtils.isAppInstalled(context, WHATSAPP_PACKAGE_NAME)) {
```
Comment (JorgeMucientes) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/details/views/OrderDetailCustomerInfoView.kt:323]: Minor np, I think the method where this new options have been added deserves some renaming. Maybe something like `showContactCustomerOptionsDialog` ? Wdyt?

**Review by JorgeMucientes — APPROVED** [review]

**Comment by JorgeMucientes** [comment]
Nice job @hichamboushaba it works as expected and code looks good!  I left a minor np you might want to check, but feel free to merge otherwise.

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/details/views/OrderDetailCustomerInfoView.kt (hunk 1 above)
Comment (hichamboushaba) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/details/views/OrderDetailCustomerInfoView.kt:323]: Good point, applied in 6e44cae43a4a40c5b90b6007307c2b2d0de9ef8d

**Pull request merged**